src/components/_product/HeroPhysical/HeroPhysical.module.scss

```diff
@@ -0,0 +1,196 @@
+.HeroPhysical {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 32px;
+
+  @media (max-width: 840px) {
+    grid-template-columns: 1fr;
+  }
```

Comment:
CSS trick: The `grid-template-columns` is set to 1fr by default. I would rather declare
```
@media (min-width: 840px) {
  grid-template-columns: 1fr 1fr;
}
```

Reply:
Some type of mobile screens can have problems with float px width, for example screen can be 840.2px wide and it will cause layout shift. Preferable to not use max and min width media query on the same page.